scylla-cdc/src/log_reader.rs

```diff
@@ -0,0 +1,213 @@
+use std::sync::Arc;
+
+use anyhow;
+use chrono::Duration;
+use core::time;
```

Comment:
Importing from `core` is a bit strange. This should be the same as `use std::time;`.